### PR TITLE
Fix Stadtforum workbench

### PR DESCRIPTION
**Pull request by xi**
Some fixups to #2746.

- When coming from the process overview, the poll listing was empty. This was fixed in 	834413f.
- The layout was improved.
- The create form was improved
- Edit was implemented

Still to do (separately):

- Handle the fact that a poll is in draft state after creation (either show something useful or do transition to participate directly)
- Add links to edit/create in the UI

### CODE REVIEW

**Comment by xi** [comment]
Note that creating/editing polls is also possible in the SDI, so implementing these features in the frontend is low-prio.

File: src/meinberlin_lib/meinberlin_lib/static/js/Packages/Meinberlin/Stadtforum/Process/Process.ts [hunk 1]
```diff
@@ -134,13 +134,13 @@ export var detailDirective = (
             }];
             scope.sort = "item_creation_date";
 
+            scope.contentType = RIProposalVersion.content_type;
```
Comment (der-john) [inline src/meinberlin_lib/meinberlin_lib/static/js/Packages/Meinberlin/Stadtforum/Process/Process.ts]: aaah. nice.

**Comment by der-john** [comment]
lgtm. I think I'm going to add an "add poll" button like in #2784 later.

**Comment by der-john** [comment]
unfortunately, ordinary users seem to lack view permissions on polls.

**Comment by xi** [comment]
> unfortunately, ordinary users seem to lack view permissions on polls.

I guess you mean that polls created in the UI do not show up? That is because they are created in draft state, as stated above. Fixing this is not urgent because it can be handled in the SDI.

File: src/meinberlin_lib/meinberlin_lib/static/js/Packages/Meinberlin/Stadtforum/Proposal/Module.ts [hunk 2]
```diff
@@ -42,6 +42,21 @@ export var register = (angular) => {
...
+            "adhRate",
+            "adhResourceUrlFilter",
+            "adhShowError",
+            "adhSubmitIfValid",
+            "adhTopLevelState",
+            "adhGetBadges",
+            "$location",
+            "$q",
+            Proposal.editDirective
+        ])
```
Comment (der-john) [inline src/meinberlin_lib/meinberlin_lib/static/js/Packages/Meinberlin/Stadtforum/Proposal/Module.ts]: General remark: There is no button in the UI to get to the edit view yet.

**Comment by der-john** [comment]
works.

**Pull request merged**